docs/development/pullrequests.rst

```diff
@@ -1,43 +1,84 @@
+  #ISSUENUMBER syntax to link it)
 
-Accepting Pull Requests
-=======================
 
-`ctapipe` maintainers must do a code review before accepting any pull
-request. The following guidelines should be used to facilitate the
-procedure:
+Note that you can include syntax-hlighted code examples by using 3 back-tics:
 
-* Perform a Scientific or Conceptual Review
+.. code-block:: markdown
+		
+   ```python
+   
+   code here
+   
+   ```
 
-  * Look at the use case for the proposed change.
-    Is it clear? If not, ask for improvement
 
-* Perform a Code Review
 
-  * Check that all functions and classes have API documentation in the
-    correct format (and check that there are no warnings generated when
-    building the docs)
-  * Check that there are at least basic unit tests for each function and class.
-  * Run all unit tests
+Accepting a Pull Request
+------------------------
 
-   * If any unit tests included in the pull request fail, ask the developer if that is normal (sometimes there are tests intended to fail until a feature is available or implemented)
-   * If any other unit tests that previously suceeded now fail, **stop** and ask the developer to find out what was broken by their change
+`ctapipe` maintainers must do a *code review* before accepting any
+pull request. During the review the reviewer can ask for changes to be
+made, and the requester can simply push them to the branch associated
+with the request and they will automatically appear (no new pull
+request needed).  The following guidelines should be used to
+facilitate the review procedure:
 
-  * Read through the new code being contributed, and see that it follows the style guidelines and that the API
-  * Check that the API (function and class definitions) is clear and easy to understand. If not, ask for it to be cleaned up
+* Perform a Scientific or Conceptual Review if the request introduces
+  new features, algorithms, or design changes
+  
+ - Look at the use case for the proposed change.
+   
+  + if the use case is missing, ask for one
+  + does it make sense? Is it connected to a goal, requirement, or specification?
+    
+* Perform a Code Review in 2 passes
+  
+  - Check that all functions and classes have API documentation in the
+    correct format (and check that there are no warnings generated
+    whenbuilding the docs)
+  - Check that there are at least basic unit tests for each function and class.
+  - Run all unit tests
+    
+   + If any unit tests included in the pull request fail, ask the
+     developer if that is normal (sometimes there are tests intended
+     to fail until a feature is available or implemented)
+   + If any other unit tests that previously suceeded now fail,
+     **stop** and ask the developer to find out what was broken by
+     their change
+     
+  - Read through the new code being contributed, and see that it
+    follows the style guidelines and that the API
+    
+   + no lines over 80 cols
```

Comment:
Most people nowadays advocate 90 characters, 80 is really restrictive.

Raymond Hattinger has a talk about this: https://www.youtube.com/watch?v=wf-BqAjZb8M